modal/app.py

```diff
@@ -112,6 +85,36 @@ def stop_fetching_inputs(self):
 assert isinstance(container_app, ContainerApp)
 
 
+async def _init_container_app(
+    client: _Client,
+    app_id: str,
+    environment_name: str = "",
+    function_def: Optional[api_pb2.Function] = None,
+):
+    """Used by the container to bootstrap the app and all its objects. Not intended to be called by Modal users."""
+    global _container_app, _is_container_app
```

Comment:
Can we not pass the container app in when we call this? I'm a little confused about the order of operations but not a big deal since it's basically consistent with the status quo.

Reply:
we don't have to, but then we'd have to have a separate function to set `_is_container_app = True`

this is fairly temporary anyway so I think let's leave it for now and optimize further later